lib/version_utils.pm

```diff
@@ -119,7 +119,8 @@ sub is_storage_ng {
 }
 
 sub is_sle12_hdd_in_upgrade {
-    return get_var('UPGRADE') && !sle_version_at_least('15', version_variable => 'HDDVERSION');
+    return (get_var('UPGRADE') || get_var('ONLINE_MIGRATION') || get_var('ZDUP') || get_var('AUTOUPGRADE'))
```

Comment:
Why not to reuse is_upgrade_test method you've introduced here as well?

Reply:
The function is_upgrade_test is defined in main_common.pm, I thought it was weird to use main_common.pm function in version_utils.pm 
I will update according to mitiao's suggestion: move is_upgrade_test to version_utils.pm and rename it to is_upgrade, then I can reuse the function here in is_sle12_hdd_in_upgrade